cmd/build-push-image/main.go

```diff
@@ -22,8 +23,10 @@ import (
 )
 
 const (
-	aquasecBin                  = "aquasec"
-	kubernetesServiceaccountDir = "/var/run/secrets/kubernetes.io/serviceaccount"
+	aquasecBin                    = "aquasec"
+	kubernetesServiceaccountDir   = "/var/run/secrets/kubernetes.io/serviceaccount"
+	scanComplianceFailureExitCode = 4
+	// scanLicenseValidationFailureExitCode = 5
```

Comment:
Shall we remove this comment or might we handle this specific exit code explicitly later on?

Reply:
Hehe, wasn't sure. I think it is valuable to know which exit codes may be returned, so I did not want to delete it.